orderbook/usecase/orderbook_usecase_test.go

```diff
@@ -35,6 +35,101 @@ func TestOrderbookUsecaseTestSuite(t *testing.T) {
 	suite.Run(t, new(OrderbookUsecaseTestSuite))
 }
 
+// Creates a new osmomath.Dec from a string
+// Panics if the string is invalid
+func (s *OrderbookUsecaseTestSuite) newDecFromStr(str string) osmomath.Dec {
+	s.T().Helper()
+	dec, err := osmomath.NewDecFromStr(str)
+	s.Require().NoError(err)
+	return dec
+}
+
+func (s *OrderbookUsecaseTestSuite) newOrder() orderbookdomain.Order {
+	s.T().Helper()
+
+	return orderbookdomain.Order{
+		TickId:         1,
+		OrderId:        1,
+		OrderDirection: "bid",
+		Owner:          "owner1",
+		Quantity:       "1000",
+		PlacedQuantity: "1500",
+		Etas:           "500",
+		ClaimBounty:    "10",
+		PlacedAt:       "1634764800000",
+	}
+}
+
+func (s *OrderbookUsecaseTestSuite) newLimitOrder() orderbookdomain.LimitOrder {
+	s.T().Helper()
+
+	return orderbookdomain.LimitOrder{
+		TickId:           1,
+		OrderId:          1,
+		OrderDirection:   "bid",
+		Owner:            "owner1",
+		Quantity:         osmomath.NewDec(1000),
+		Etas:             "500",
+		ClaimBounty:      "10",
+		PlacedQuantity:   osmomath.NewDec(1500),
+		PlacedAt:         1634,
+		Price:            s.newDecFromStr("1.000001000000000000"),
+		PercentClaimed:   s.newDecFromStr("0.333333333333333333"),
+		TotalFilled:      s.newDecFromStr("600"),
+		PercentFilled:    s.newDecFromStr("0.400000000000000000"),
+		OrderbookAddress: "someOrderbookAddress",
+		Status:           "partiallyFilled",
+		Output:           s.newDecFromStr("1499.998500001499998500"),
+		QuoteAsset:       orderbookdomain.Asset{Symbol: "ATOM", Decimals: 6},
+		BaseAsset:        orderbookdomain.Asset{Symbol: "OSMO", Decimals: 6},
+	}
+}
+
+func (s *OrderbookUsecaseTestSuite) newTick(effectiveTotalAmountSwapped string, unrealizedCancels int64, direction string) orderbookdomain.OrderbookTick {
```

Comment:
This one makes sense to be a helper IMO - nice work!

However, could you please add some comments to make this more grokable?

Reply:
Added some docs/comments :+1: